spec/spec_helper.rb

```diff
@@ -102,23 +101,16 @@ def osi_approved_licenses
 
 def fsf_approved_licenses
   SpecHelper.fsf_approved_licenses ||= begin
-    url = 'https://www.gnu.org/licenses/license-list.en.html'
-    doc = Nokogiri::HTML(open(url).read)
-    list = doc.css('.green dt')
+    url = 'https://wking.github.io/fsf-api/licenses-full.json'
+    object = JSON.parse(open(url).read)
     licenses = {}
-    list.each do |license|
-      a = license.css('a').find { |link| !link.text.nil? && !link.text.empty? && link.attr('id') }
-      next if a.nil?
-      id = a.attr('id').downcase
-      name = a.text.strip
-      licenses[id] = name
-    end
-
-    # FSF approved the Clear BSD, but doesn't use its SPDX ID or Name
-    if licenses.keys.include? 'clearbsd'
-      licenses['bsd-3-clause-clear'] = licenses['clearbsd']
```

Comment:
To help with review, this rename is handled in the mock API:

```console
$ curl -s https://wking.github.io/fsf-api/clearbsd.json | jq .identifiers.spdx
[
  "BSD-3-Clause-Clear"
]
```

The MIT / Expat [and other mappings][1] are also handled in the mock API.

[1]: https://github.com/wking/fsf-api/blob/695b1fdd58691a4dfee80b2569cda81fbd9a1739/pull.py#L94-L215